ios/PurchasesHybridCommon/ObjCAPITester/RCHybridCommonAPITest.m

```diff
@@ -98,7 +100,7 @@ - (void)testAPI {
 
     [RCCommonFunctionality setFinishTransactions:NO];
     [RCCommonFunctionality checkTrialOrIntroductoryPriceEligibility:@[@""]
-                                                    completionBlock:^(NSDictionary<NSString *,RCIntroEligibility *>
+                                                    completionBlock:^(NSDictionary<NSString *, NSObject *>
```

Comment:
this was actually wrong in the signature in the objc version - it doesn't actually return an `RCIntroEligibility`, it returns a dict with 
```
"status": IntroEligibilityStatus (an enum),
"description": String
```